utils/mqttled/Makefile

```
@@ -0,0 +1,57 @@
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mqttled
+PKG_VERSION:=0.1.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=mqttled
+PKG_HASH:=20f46b7114b6ddace6e0faaaa078640f609c3d495e4ccff8d8caeb77ac5418f7
+
+PKG_MAINTAINER:=Tom Grime <tom.grime@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include ../../lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/mqttled
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=LED Control from MQTT 
+  URL:=https://github.com/trevortrevor/mqttled/
+  DEPENDS:= \
+    +python3-paho-mqtt \
+	+python3-uci \
+	+python3-idna \
+	+python3-jsonpath-ng \
+	+python3-yaml \
+	+python3-schema \
+	+python3-netifaces \
+	+python3-logging \
+	+python3-urllib \
+	+python3-asyncio
+
+endef
+
+define Package/mqttled/description
+	Exposes the LEDs of your device to an MQTT broker with optional Homeassistant auto discovery
+endef
+
+define Py3Package/mqttled/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(CP) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/mqttled.init $(1)/etc/init.d/mqttled
+	$(INSTALL_DIR) $(1)/etc/config
+	$(INSTALL_DATA) ./files/mqttled $(1)/etc/config/mqttled
```

```suggestion
	$(INSTALL_DATA) ./files/mqttled.config $(1)/etc/config/mqttled
```

Suggest to rename the sample config file in the package directory.